src/Microsoft.VisualStudio.Editors/PropPages/BuildPropPage.resx

```diff
@@ -975,6 +975,12 @@
   <data name="cboWarningLevel.Items4" xml:space="preserve">
     <value>4</value>
   </data>
+  <data name="cboWarningLevel.Items5" xml:space="preserve">
+    <value>5</value>
+  </data>
+  <data name="cboWarningLevel.Items6" xml:space="preserve">
```

Comment:
Can we give this value a special name or explain the value so that it doesn't look odd? If I hadn't read the specification I would have thought this was a bug.